core/services/chainlink/config.go

```diff
@@ -129,10 +129,6 @@ func (s *Secrets) SetFrom(f *Secrets) (err error) {
 		err = multierr.Append(err, config.NamedMultiErrorList(err1, "Database"))
 	}
 
-	if err2 := s.Explorer.SetFrom(&f.Explorer); err2 != nil {
```

Comment:
/nit and a matter of preference - in the future when I revisit this file I'd spend some unnecessary time answering the question "why is err2 skipped?". If you'd like, perhaps update the ordering here

Reply:
@patrickhuie19 just pushed a commit for this